lessons/Algebra2/linear2-building-models/langs/en-us/index.adoc

```diff
@@ -115,15 +126,24 @@ Building on prior knowledge of linear functions, students learn to find the line
 This scatter plot appears to show a positive, linear relationship: +
 _States with higher percentages of college graduates tend to have higher median household incomes._
 }
-@ifnotslide{@center{@image{images/college-v-income.png}}}
+@ifnotslide{@center{@image{images/college-v-income.png, 450}}}
 
 @slidebreak
 
 @QandA{
 @Q{What do you notice about the @vocab{Form} of this scatter plot? What pattern do you see?}
```

Comment:
Now that the correlations lesson isn't required, I don't think we should use the vocabulary word `form` here. We also don't _need_ it! All we really want to ask is what shape the relationship looks like, right?